### DISCUSSION TITLE
ESP32-S2 internal DAC support

### DISCUSSION THREAD

**Discussion by Garrylua** [pschatzmann/arduino-audio-tools#1590] [Q&A] [answered]
Hi,
I would like to use the arduino-audio-tools library to stream audio to the internal DACs on the ESP32-S2.
On the standard ESP32 this is done with I2S, but that is not the case for the ESP32-S2. I think some other kind of internal DMA is used.

I think I saw in another discussion that board sdk/driver version 3.0.0 would be required. I note that 3.0.1 is now available.
Does that mean that the ESP32-S2 is now supported, or will be in the near future? 
Given my lack of experience I would also love to see some relevant examples.
Thanks.

**Comment by pschatzmann** [comment, marked as answer]
Sure, you can use the internal DAC or ADC (if your processor supports it) with the help of the AnalogAudio Class if you use a core version that supports IDF >= 5. 

So Arduino ESP32 3.0.1 should work

**Reply by Garrylua** [reply]
Thanks, 
I'll give it a try.
